src/jit/target.h

```diff
@@ -1884,11 +1892,58 @@ inline bool         genIsValidDoubleReg(regNumber reg)
 #endif // defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
 
 /*****************************************************************************
+*
+*  Returns true if our target architecture uses a fixed return buffer register
+*/
+inline bool         hasFixedRetBuffReg()
+{
+    // Disable this until the VM changes are also enabled
+#if 0 //def _TARGET_ARM64_
+    return true;
+#else
+    return false;
+#endif
+}
+
+/*****************************************************************************
+*
+*  Returns the regNumber for the fixed return buffer 
+*/
+inline regNumber    theFixedRetBuffReg()
+{
+    assert(hasFixedRetBuffReg());   // This predicate should be checked before calling this method
+#ifdef _TARGET_ARM64_
+    return REG_ARG_RET_BUFF;
+#else
+    return REG_NA;
+#endif
+}
+
+/*****************************************************************************
+*
+*  Returns the argNum for the fixed return buffer 
+*/
+inline unsigned     theFixedRetBuffArgNum()
+{
+    assert(hasFixedRetBuffReg());   // This predicate should be checked before calling this method
+#ifdef _TARGET_ARM64_
+    return RET_BUFF_ARGNUM;
+#else
+    return (unsigned) -1;
+#endif
```

Comment:
This should be BAD_VAR_NUM; is not defined this way because BAD_VAR_NUM isn't visible here (I would think that it should be, since it's in jit.h)? If so, you should add a comment to that effect.